frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,44 @@
+.errorMessage {
+  padding: 0;
```

Comment:
A default `p` element doesn't have any `padding` applied so there's no need for this declaration.